front/app/modules/commercial/matomo/matchPath.ts

```diff
@@ -29,23 +35,19 @@ function compilePath(path, options) {
 /**
  * Public API for matching a URL pathname to a path.
  */
-function matchPath(pathname, options = {} as any) {
-  if (typeof options === 'string' || Array.isArray(options)) {
-    // eslint-disable-next-line no-param-reassign
-    options = { path: options };
-  }
```

Comment:
matchPath is only used in 1 place and options is never a string nor array, so this could be deleted. You can doublecheck if I'm not overlooking something. 🤔